README.md

```diff
@@ -366,7 +366,7 @@ isSpace({}); // returns false
 
 Parameter: A space
 
-Returns: The root space associated with the given space.
+Returns: The the deepest root space associated with the given space.
```

Comment:
Do you mean "newest"?

Reply:
I threw that in to somehow let the future readers know this is not just an awkward name for what should have been a `parentOf()` function and that it goes all the way up the chain. I guess it will have to be reworded to make it clearer.